src/init.c

```diff
@@ -45,6 +45,8 @@ extern void _event_trigger_fini(void);
 extern void _conn_plain_init();
 extern void _conn_plain_fini();
 
+extern void _constraint_aware_append_init(void);
```

Comment:
I'd prefer including the .h and not doing this. This type of redeclaration is something we should move away from.

Reply:
changed